pandas/tests/arrays/string_/test_string.py

```diff
@@ -272,34 +406,49 @@ def test_reduce_missing(skipna):
 
 
 @td.skip_if_no("pyarrow", min_version="0.15.0")
-def test_arrow_array():
+def test_arrow_array(dtype, request):
     # protocol added in 0.15.0
     import pyarrow as pa
 
-    data = pd.array(["a", "b", "c"], dtype="string")
+    if dtype == "arrow_string":
+        reason = (
+            "TypeError: Argument 'other' has incorrect type "
+            "(expected pyarrow.lib.ChunkedArray, got pyarrow.lib.StringArray)"
+        )
+        mark = pytest.mark.xfail(reason=reason)
+        request.node.add_marker(mark)
+
+    data = pd.array(["a", "b", "c"], dtype=dtype)
     arr = pa.array(data)
     expected = pa.array(list(data), type=pa.string(), from_pandas=True)
     assert arr.equals(expected)
 
 
 @td.skip_if_no("pyarrow", min_version="0.15.1.dev")
-def test_arrow_roundtrip():
+def test_arrow_roundtrip(dtype, dtype_object):
     # roundtrip possible from arrow 1.0.0
     import pyarrow as pa
 
-    data = pd.array(["a", "b", None], dtype="string")
+    data = pd.array(["a", "b", None], dtype=dtype)
     df = pd.DataFrame({"a": data})
     table = pa.table(df)
     assert table.field("a").type == "string"
     result = table.to_pandas()
-    assert isinstance(result["a"].dtype, pd.StringDtype)
+    assert isinstance(result["a"].dtype, dtype_object)
     tm.assert_frame_equal(result, df)
     # ensure the missing value is represented by NA and not np.nan or None
     assert result.loc[2, "a"] is pd.NA
 
 
-def test_value_counts_na():
-    arr = pd.array(["a", "b", "a", pd.NA], dtype="string")
+def test_value_counts_na(dtype, request):
+    if dtype == "arrow_string":
+        reason = (
+            "AttributeError: 'ArrowStringArray' object has no attribute 'value_counts'"
```

Comment:
You actually added this now?

Reply:
failure is now `TypeError: boolean value of NA is ambiguous`

will change message in first instance. was planning to address the failures in this test module as a follow-on and only activated the tests here to lockdown the passing tests for this PR.